### ISSUE TITLE
open_directory with file select

### ISSUE THREAD

**Issue by streanger** [streanger/duplicate#1]
`open_directory` with file select:
https://stackoverflow.com/questions/13680415/how-to-open-explorer-with-a-specific-file-selected

```cmd
explorer.exe /select,"C:\Folder\subfolder\file.txt"
```

related part of code:
https://github.com/streanger/duplicate/blob/f665bf6cfb9be7e56bebbae70e5921a027c30bb4/duplicate/main.py#L408-L424

**Comment by streanger** [comment]
possible fix:
```python
import subprocess
file_path = ...
subprocess.Popen(f'explorer /select,"{file_path}"')
```

**Comment by streanger** [comment]
fixed in e7c46b3a41d9de7e982c6368d7fa49ba4065c049
